README.md

```diff
@@ -367,12 +367,7 @@ type t = {
 
 let rand_str len =
   Random.self_init ();
```

Comment:
```suggestion
let _ = Random.self_init ()

let rand_str len =
```

This might work as well. Init the random state when the module is loaded, so it only happens once